ingestion-data/staging/dataset-config/tornadoes-2024-dow-refl-greenfield.json

```diff
@@ -0,0 +1,79 @@
+{
+    "collection": "tornadoes-2024-dow-refl-greenfield",
+    "title": "DOW7 Reflectivity (Greenfield, IA)",
+    "description": "Horizontal Reflectivity values (dBZ) collected by DOW7 from the Greenfield, Iowa EF-4 tornado on May 21, 2024 at 20:41:08.",
+    "license": "CC0-1.0",
+    "is_periodic": true,
+    "time_density": "day",
+    "spatial_extent": {
+        "xmin": -94.5,
+        "ymin": 41.275,
+        "xmax": -94.4,
+        "ymax": 41.325
+    },
+    "temporal_extent": {
+        "startdate": "2024-05-21T00:00:00Z",
+        "enddate": "2024-05-21T23:59:59Z"
+    },
+    "sample_files": [
+        "s3://veda-data-store-staging/2024tornadoes/DOW_Greenfield_REFL_cog_v2_2024-05-21.tif"
+    ],
+    "providers": [
+        {
+            "name": "NASA VEDA",
+            "url": "https://www.earthdata.nasa.gov/dashboard/",
+            "roles": [
+                "host"
+            ]
+        }
+    ],
+    "assets": {
+        "thumbnail": {
+            "title": "Thumbnail",
+            "description": "Photo by [Ryan McGinnis](https://static.wikia.nocookie.net/stormchasers/images/1/1b/DOW.jpg/revision/latest?cb=20100115235828) (DOW6 parked in front of a tornadic mesocyclone in the Plains)",
+            "href": "https://thumbnails.openveda.cloud/tornadoes-2024-dow-background.jpg",
```

Comment:
```suggestion
            "description": "Photo by [Ryan McGinnis](https://static.wikia.nocookie.net/stormchasers/images/1/1b/DOW.jpg/revision/latest?cb=20100115235828) (DOW6 parked in front of a tornadic mesocyclone in the Plains)",
            "href": "https://thumbnails.openveda.cloud/tornadoes-2024-dow-background.jpg",
```

Reply:
This one got fixed outside of the PR so I am resolving